### PR TITLE
Refactor/update dependency

### PR DESCRIPTION
**Pull request by Hepik**

### CODE REVIEW

File: src/app/(site)/(root)/_components/ReviewList/reviewList.tsx [hunk 1]
```diff
@@ -62,9 +62,6 @@ export const ReviewList = () => {
     <div className="max-w-[1200px]">
       {data?.reviews.map((review: any, index: number) => (
         <div key={review.id} className="flex items-center space-x-2">
-          <span className="font-bold text-gray-500">
-            {(page - 1) * limit + index + 1}.
-          </span>
```
Comment (schistostega) [inline src/app/(site)/(root)/_components/ReviewList/reviewList.tsx:67]: You have just removed the span. Are You sure that it is ok to delete?

File: src/app/layout.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en">
+    <html lang="en" suppressHydrationWarning={true}>
```
Comment (schistostega) [inline src/app/layout.tsx:21]: I don't thik that warning suppression is a good idea. Could You try to solve the hydration issues?

**Review by schistostega — CHANGES_REQUESTED** [review]

File: src/app/(site)/(root)/_components/ReviewList/reviewList.tsx (hunk 1 above)
Comment (Hepik) [reply src/app/(site)/(root)/_components/ReviewList/reviewList.tsx:67]: This is unnecessary span

File: src/app/layout.tsx (hunk 2 above)
Comment (Hepik) [reply src/app/layout.tsx:21]: The next-themes causes those warnings, and i can`t fix them directly.

**Review by schistostega — APPROVED** [review]

**Pull request merged**